mlflow/R/mlflow/tests/testthat/test-tracking-runs.R

```diff
@@ -228,6 +228,41 @@ test_that("with() errors when not passed active run", {
   )
 })
 
+test_that("mlflow_search_runs() works", {
+  mlflow_clear_test_dir("mlruns")
+  with(mlflow_start_run(), {
+    mlflow_log_metric("test", 10)
+  })
+  with(mlflow_start_run(), {
+    mlflow_log_metric("test", 20)
+  })
+  expect_equal(nrow(mlflow_search_runs(experiment_ids = list("0"))), 2)
```

Comment:
Do we test search  with inferred experiment id?
If we want to support single experiment id argument, we should probably add a test case for that as well.

Reply:
Yep, we test `mlflow_search_runs()` without arguments (resulting in an inferred experiment) in this test, see [here](https://github.com/mlflow/mlflow/pull/1310/files/e7261526fd7101b67c1a55942fb8d991f609e854#diff-0b452f166f590473c5bf4e7aadb67595R244). Good point on testing passing a single experiment ID, added a test case for that.